API/TreeData/TreeDataMock.cs

```diff
@@ -0,0 +1,149 @@
+        public string StemTag { get; set; }
+        public int ErrorCode { get; set; }
+        public string Error { get; set; }
+        public TreeResponse (TreeRequest tree, int ecode, string error){
+            Subquadrat = tree.Subquadrat;
+            Tag = tree.Tag;
+            StemTag = tree.StemTag;
+            ErrorCode = ecode;
+            Error = error;
+        }
+    }
+    public static class TreeDataMock
+    {
+        private static string connStr = System.Environment.GetEnvironmentVariable("MySQLConnection", EnvironmentVariableTarget.Process);
+        private static MySqlConnection conn = new MySqlConnection(connStr);
+
+        private static ArrayList QueryTreeDB(string sql){
+            var LoadingResponse = new ArrayList();
+            try
+            {
+                Console.WriteLine("Connecting to MySQL...");
+                conn.Open();
+
+                MySqlCommand cmd = new MySqlCommand(sql, conn);
+                MySqlDataReader rdr = cmd.ExecuteReader();
+                
+                while (rdr.Read())
+                    LoadingResponse.Add(new TreeStorage().MapSQL(rdr));
+                rdr.Close();
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.ToString());
+                //return new FailedObjectResult("Failed to process request");
+            }
+            conn.Close();
+            Console.WriteLine("Done.");
+            
+            return LoadingResponse;
+        }
+
+        [FunctionName("GetTrees")]
+        public static IActionResult GetTrees(
+            [HttpTrigger(AuthorizationLevel.Function, "get", Route = "TreeData")] HttpRequest req,
+            ILogger log)
+        {
+            log.LogInformation("C# HTTP trigger function processed a request.");
+
+            ArrayList LoadingResponse = QueryTreeDB("select TempID, QuadratName, Tag, StemTag, Mnemonic as SpCode, DBH, Codes, HOM, ExactDate, x, y, PlotID, CensusID, Errors from tempnewplants");
+
+            return new OkObjectResult(JsonConvert.SerializeObject(LoadingResponse));
+        }
+
+        [FunctionName("PostTrees")] 
+        public static async Task<IActionResult> PostTrees(
+            [HttpTrigger(AuthorizationLevel.Function, "post", Route = "TreeData")] HttpRequest req,
+            ILogger log)
+        {
+            log.LogInformation("C# HTTP trigger function processed a request.");
+
+            string requestBody = await new StreamReader(req.Body).ReadToEndAsync();
+            var data = JsonConvert.DeserializeObject<TreeRequest[]>(requestBody);
+
+            ArrayList LoadingResponse = QueryTreeDB("select TempID, QuadratName, Tag, StemTag, Mnemonic as SpCode, DBH, Codes, HOM, ExactDate, x, y, PlotID, CensusID, Errors from tempnewplants");
+
+            var TreeResponse = new ArrayList();
+            foreach(var Tree in data){
+                if(Tree.Codes.Contains("at") && Tree.WasDead(LoadingResponse)){

```

Comment:
See me above comments using `IsAlive` and `IsDead` here

Reply:
Fixed in https://github.com/ForestGeoHack/ForestGEO/pull/12/commits/0543202ea320047257a303f345157882127330c9